README.md

```diff
@@ -93,7 +93,7 @@ While Ecoute provides real-time transcription and response suggestions, there ar
 
 **Whisper Model**: If the --api flag is not used, we utilize the 'tiny' version of the Whisper ASR model, due to its low resource consumption and fast response times. However, this model may not be as accurate as the larger models in transcribing certain types of speech, including accents or uncommon words.
 
-**Language**: If you are not using the --api flag the Whisper model used in Ecoute is set to English. As a result, it may not accurately transcribe non-English languages or dialects. We are actively working to add multi-language support to future versions of the program.
+**Language**: If you are not using the --api flag the Whisper model used in Ecoute is set to English. As a result, it may not accurately transcribe non-English languages or dialects. The multi-language support works with local models only and is not compatible with --api.
```

Comment:
Sry for asking for this change! --api has multi-language support now so we can get rid of this limitation altogether.